.jenkins/Jenkinsfile

```diff
@@ -98,7 +140,15 @@ def win2016DebugCrossPlatform() {
             def oetoolsWincp = docker.build("oetools-wincp", "-f .jenkins/Dockerfile .")
             oetoolsWincp.inside {
                 timeout(15) {
-                    sh './scripts/test-build-config -p SGX1FLC -b Debug --compiler=clang-7'
+                    dir('build') {
+                        withEnv(["CC=clang-7","CXX=clang++-7","OE_SIMULATION=1"]) {
+                            sh """
+                            cmake .. -DCMAKE_BUILD_TYPE=Debug -DUSE_DEBUG_MALLOC=OFF
+                            make
+                            ctest --output-on-failure
```

Comment:
Sorry for the late review - but you don't really need the ctest. The OE_SIMULATION flag does not matter here as we only need to build the ELF enclaves on Linux. It was overkill to use the test-build-config script for this step but was probably used for convenience.

Reply:
Oh, it makes sense! We will address this into a future PR for the Jenkinsfile.

Thank-you for the review @anitagov!

Reply:
We will be fixing this into commit https://github.com/Microsoft/openenclave/pull/1407/commits/ab07acab7510e9ea6437ddda25ec3c7d195e0573 as part of the PR https://github.com/Microsoft/openenclave/pull/1407.